exporting/opentsdb/opentsdb.c

```diff
@@ -240,26 +258,26 @@ int format_dimension_stored_opentsdb_telnet(struct instance *instance, RRDDIM *r
 }
 
 /**
- * Prepare an HTTP message for OpenTSDB HTTP connector
+ * Ppepare HTTP header
```

Comment:
Typo error, instead `Prepare` we have `Ppepare`.

Reply:
Fixed.